request-manager/request/manager.go

```diff
@@ -493,8 +498,102 @@ func (m *manager) GetWithJC(requestId string) (proto.Request, error) {
 	return req, nil
 }
 
+func (m *manager) Find(filter proto.RequestFilter) ([]proto.Request, error) {
+	ids, err := m.findIDs(filter)
+	if err != nil {
+		return []proto.Request{}, fmt.Errorf("Error finding requests matching filter: %w", err)
+	}
+
+	requests := make([]proto.Request, 0, len(ids))
+	for _, id := range ids {
+		req, err := m.Get(id)
+		if err != nil {
+			return []proto.Request{}, fmt.Errorf("Error retrieving request %s: %w", id, err)
+		}
+		requests = append(requests, req)
+	}
+
+	return requests, nil
+}
+
 // ------------------------------------------------------------------------- //
 
+// Return request IDs matching the filter, ordered by create time + request ID.
```

Comment:
Why not get the full request here so that we don't have to run two queries for each one?

Reply:
No great reason, I was just being a bit lazy.